builds/posix/prefix.freebsd_powerpc64el

```diff
@@ -0,0 +1,31 @@
+# The contents of this file are subject to the Interbase Public
+# License Version 1.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy
+# of the License at http://www.Inprise.com/IPL.html
+#
+# Software distributed under the License is distributed on an
+# "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either express
+# or implied. See the License for the specific language governing
+# rights and limitations under the License.
+#
+# The Original Code was created by Inprise Corporation
+# and its predecessors. Portions created by Inprise Corporation are
+#
+# Copyright (C) 2000 Inprise Corporation
+# All Rights Reserved.
+# Contributor(s): ______________________________________.
+# Start of file prefix.freebsd_powerpc64el:	$(VERSION)	@PLATFORM@
+#
+# 2 Oct 2002, Nickolay Samofatov - Major Cleanup
+
+EXE_LINK_OPTIONS := $(LDFLAGS) $(THR_FLAGS) $(UNDEF_FLAGS) $(LIB_PATH_OPTS) -Wl,-rpath,../gen/firebird/lib
+
+COMMON_FLAGS=-DFREEBSD -DPPC64EL -pipe -MMD -fPIC -fsigned-char
+PROD_FLAGS=$(COMMON_FLAGS) -O3 -g -fno-builtin -Wno-deprecated
+DEV_FLAGS=$(COMMON_FLAGS) -ggdb -p -Wall -Wno-non-virtual-dtor
+
+# These files are generated incorrectly (e.g. array.epp => array.cpp)
+%/array.o %/blob.o %/alice_meta.o %/restore.o %/backup.o: COMMON_FLAGS += -Wno-narrowing
+%/OdsDetection.o %/dba.o: COMMON_FLAGS += -Wno-narrowing
+
+CXXFLAGS := $(CXXFLAGS) -std=c++11
```

Comment:
What a reason to change default for master branch c++17 to c++11? I hardly understand how does it build at all cause in alloc.h we use c++14 specific std::enable_if_t, see https://en.cppreference.com/w/cpp/types/enable_if

Reply:
Ah, I based this patch on the Firebird 4 that we have in our ports tree and some other prefix files used std=c++11.